docs/_snippets/document-progress.mdx

```diff
@@ -0,0 +1,41 @@
+<ResponseField name="download" type="Progress">
+  <Expandable title="Download Properties">
+    <ResponseField name="status" type="string" required>
+      The status of querying document references across HIEs, either `processing` or `completed`.
+    </ResponseField>
+
+    <ResponseField name="total" type="number">
+      The total number of documents to be queried.
+    </ResponseField>
+
+    <ResponseField name="successful" type="number">
+      The number of documents successfully downloaded.
+    </ResponseField>
+
+    <ResponseField name="errors" type="number">
+      The number of documents that failed to download.
+    </ResponseField>
+
+  </Expandable>
+</ResponseField>
+
+<ResponseField name="convert" type="Progress">
+  <Expandable title="Convert Properties">
+    <ResponseField name="status" type="string" required>
+      The status of converting CCDA documents to FHIR, either `processing` or `completed`.
+    </ResponseField>
+
+    <ResponseField name="total" type="number">
+      The total number of documents to be converted.
+    </ResponseField>
+
+    <ResponseField name="successful" type="number">
+      The number of documents successfully converted.
+    </ResponseField>
+
+    <ResponseField name="errors" type="number">
+      The number of documents that failed to convert.
```

Comment:
To consider:

In both `error` cases - should we add to the docs and instruct the dev to take an action, or at least give some info about *why* something could fail?

From the dev perspective, the first thing I'd think if I saw an error count would be "why" and "what can I do to get this number to 0".

Now that I think about it, should we provide more details here? ie make this an object containing the doc ref ID so the dev knows which file failed, if any?

Right now we raise a flag, but the dev can't do anything actionable except retry.

Any thoughts on this @Orta21 @leite08 ? Can raise a follow up issue if needed for sure (not critical right now)

Reply:
I like the idea to improve on this, not sure exactly what _actionable_ information we could provide our cxs, though. So yeah, I think a follow-up ticket to figure that out and implement would be ideal.

Reply:
https://github.com/metriport/metriport-internal/issues/779